### PR TITLE
Sign message with WalletConnect

### PR DESCRIPTION
**Pull request by amacar**
[ch5342]

### CODE REVIEW

File: common/v2/components/WalletList/WalletButton.tsx [hunk 1]
```diff
@@ -78,26 +77,29 @@ const WalletLabel = styled.div`
...
-        <WalletLabel>{name}</WalletLabel>
-      </WalletButtonWrapper>
-    );
-  }
-
-  private handleClick = () => {
-    this.props.onClick(this.props.walletType);
-  };
-}
+export const WalletButton = (props: Props) => {
```
Comment (vilsbole) [inline common/v2/components/WalletList/WalletButton.tsx]: Can we add a `story` for this please?

File: common/v2/services/WalletService/walletconnect/useWalletConnect.ts [hunk 2]
```diff
@@ -86,7 +87,7 @@ export function useWalletConnect() {
       } else {
         return service
           .sendTx(tx)
-          .then((txHash: ITxData) => {
+          .then((txHash: string) => {
```
Comment (vilsbole) [inline common/v2/services/WalletService/walletconnect/useWalletConnect.ts]: We now have a `ITxSigned` and `ITxHash` `Brand` types and type guards to deal with a response from a wallet. It's a slight improvement over using a simple `string`.
https://github.com/MyCryptoHQ/MyCrypto/blob/85c493da74a334806eb476bbc391074e6566aff3/common/v2/utils/typeGuards.ts#L8

Comment (amacar) [reply common/v2/services/WalletService/walletconnect/useWalletConnect.ts]: Changed to ITxHash, thanks for the info!

File: common/v2/components/WalletList/WalletButton.tsx (hunk 1 above)
Comment (amacar) [reply common/v2/components/WalletList/WalletButton.tsx]: Added story with two states (enabled/disabled).

**Pull request merged**